config/jobs/kubernetes/sig-cloud-provider/gcp/gcp-gce.yaml

```diff
@@ -622,6 +622,110 @@ periodics:
     testgrid-alert-email: kubernetes-sig-testing-alerts@googlegroups.com, release-team@kubernetes.io
     description: Uses kubetest to run e2e tests (-Slow|Serial|Disruptive|Flaky|Feature) against a cluster (ubuntu based and uses containerd) created with cluster/kube-up.sh
 
+- interval: 120m
```

Comment:
this file isn't the right place to add kOps jobs. they must be with the rest of the kOps jobs.

Reply:
It is not a kops job, I'm using kops to bootstrap the cluster.

Reply:
@neolit123 see https://github.com/kubernetes/kubernetes/issues/78995#issuecomment-1707025083

we don't want to add support for arm64 to current cluster/kube-up.sh